spec/models/activity_spec.rb

```diff
@@ -1,9 +1,12 @@
 require 'rails_helper'
 
 describe Activity do
+
   it { should have_and_belong_to_many(:users) }
   
-  it 'is not being saved without a name'
-  it 'will not be saved if there is a activity with the same name '
+  subject { create(:activity) }
```

Comment:
i thought so too at first. but without that line, I get this when running rspec:
```ruby
Failures:

  1) Activity should require case sensitive unique value for name
     Failure/Error: it { should validate_uniqueness_of(:name) }
     NoMethodError:
       undefined method `primary' for #<ActiveRecord::ConnectionAdapters::Column:0x00000008c43bc0>
     # ./spec/models/activity_spec.rb:8:in `block (2 levels) in <top (required)>'
     # ./spec/rails_helper.rb:59:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:58:in `block (2 levels) in <top (required)>'
```